### DISCUSSION TITLE
Newsletters scope

### DISCUSSION THREAD

**Discussion by mathiazom** [simple-login/app#956] [Q&A] [answered]
Hi, love the service :heart: 

Does the newsletter only include feature updates, or general announcements like the Proton acquisition?

Just realized I had turned off newsletters, which probably explains why I missed the Proton news :upside_down_face:

![image](https://user-images.githubusercontent.com/24361490/167100480-8ce0f803-24d3-4344-891c-2432031dd01c.png)

**Comment by nguyenkims** [comment, marked as answer]
@mathiazom We haven't sent out newsletter for a while and the Proton news will be indeed included in the next newsletter :)
